app/src/main/java/com/example/levi9application/common/Common.kt

```diff
@@ -31,4 +42,16 @@ object Common {
         return CocktailRepo(apiService)
     }
 
+    @Provides
+    @Singleton
```

Comment:
Please rename the methods to provideSOMETHING. When using DI all methods should be named in a way so that the name starts with provide and then we add what kind of object we are providing. Do this for all other methods inside Common.kt file